src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -242,24 +255,19 @@ trait ClusterComponent extends LeoComponent {
         .update((status.toString, hostIp.map(_.value), Timestamp.from(Instant.now)))
     }
 
-    def setToRunning(id: Long, hostIp: IP): DBIO[Int] = {
-      updateClusterStatusAndHostIp(id, ClusterStatus.Running, Some(hostIp))
-    }
-
-    def setToStopping(id: Long): DBIO[Int] = {
-      updateClusterStatusAndHostIp(id, ClusterStatus.Stopping, None)
+    def updateAsyncClusterCreationFields(initBucket: Option[GcsPath],
+                                         serviceAccountKey: Option[ServiceAccountKey],
+                                         cluster: Cluster): DBIO[Int] = {
+      clusterQuery.filter { _.id === cluster.id }
+        .map(c => (c.initBucket, c.serviceAccountKeyId, c.googleId, c.operationName, c.stagingBucket, c.dateAccessed))
+        .update(initBucket.map(_.toUri), serviceAccountKey.map(_.id.value), cluster.googleId,
+          cluster.operationName.map(_.value), cluster.stagingBucket.map(_.value), Timestamp.from(Instant.now))
```

Comment:
These are the `CLUSTER` table fields that are now filled in *after* `PUT /api/cluster/{googleProject}/{clusterName}` returns.